website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -133,6 +157,13 @@ the same port as in the following example.
 $ consul connect envoy -sidecar-for db -admin-bind localhost:19001
 ```
 
+The mesh gateway Envoy process can be started with.
```

Comment:
Each example might deserve a header to distinguish it from the previous example.